Makefile

```diff
@@ -44,24 +44,24 @@ pyavd-editable-install: ## Build and install PyAVD as editable
 #########################################
 
 .PHONY: sanity
-sanity: sanity-info sanity-lint sanity-import ## Run ansible-test sanity validation.
+sanity: sanity-info sanity-lint sanity-import ## Run ansible-test sanity validation. Requires docker.
```

Comment:
Is this still correct? Since you changed the default to venv.

Reply:
this target runs all 3 tragets sanity-info sanity-lint sanity-import and  `sanity-info` requires docker. I will also update this and the document and makefile.

Reply:
nvm this comment, need more coffee this morning you are correct that we no longer require docker to run this!

Reply:
resolved with [69afeed](https://github.com/aristanetworks/avd/pull/3766/commits/69afeed98e3178895c8c616c074ff8bdd3a0f283)